orchestrator/src/test/java/com/scottlogic/datahelix/generator/orchestrator/cucumber/testframework/utils/CucumberFileReader.java

```diff
@@ -19,20 +19,19 @@
 import com.scottlogic.datahelix.generator.profile.reader.FileReader;
 
 import javax.inject.Inject;
+import java.io.File;
 
 public class CucumberFileReader extends FileReader {
     private final CucumberTestState testState;
 
     @Inject
     public CucumberFileReader(CucumberTestState testState) {
-        super("");
         this.testState = testState;
     }
 
     @Override
-    public DistributedList<String> listFromMapFile(String file, String key) {
-        return DistributedList.uniform(testState.getValuesFromMap(file, key));
+    public DistributedList<String> listFromMapFile(File file, String key) {
+        return DistributedList.uniform(testState.getValuesFromMap(file.getName(), key));
```

Comment:
if I use to pass in a filename of "examples/actornames/profile.json", does this give me the value we would expect (or just "profile.json") ?

Reply:
it will now give `profile.json`. The import files need to be 'relative' to the profile directory.

Need to check - this may be relative to the original profile file.

Reply:
I've added in some tests to prove the behaviour